### PR TITLE
Feature couchpotato

### PR DESCRIPTION
**Pull request by DariusIII**
Addresses issue #2141 

Changes made by this pull request.
##

### CODE REVIEW

File: www/themes/Alpha/templates/movies.tpl [hunk 1]
```diff
@@ -226,6 +216,13 @@
 											<div class="icon icon_sab float-right" title="Send to my Queue"></div>
 										{/if}
 										<div class="icon icon_cart float-right" title="Add to Cart"></div>
+										{if $cpurl != '' && $cpapi != ''}
+											<a
+													id="guid{$mguid[$m@index]}"
```
Comment (b3rs3rk) [inline www/themes/Alpha/templates/movies.tpl]: Indent too big?

File: www/themes/Alpha/templates/upcoming.tpl [hunk 2]
```diff
@@ -55,9 +55,7 @@
 							{if $cpurl != '' && $cpapi != ''}
 								<a
 										class="sendtocouch"
-										target="blackhole"
-										href="javascript:;"
-										rel="{$cpurl}/api/{$cpapi}/movie.add/?identifier=tt{$result->alternate_ids->imdb}&title={$result->title}" name="CP{$result->alternate_ids->imdb}" title="Add to CouchPotato"><img
+										title="Add to CouchPotato"><img
 											src="{$smarty.const.WWW_TOP}/themes/shared/img/icons/couch.png">
```
Comment (b3rs3rk) [inline www/themes/Alpha/templates/upcoming.tpl]: Move open <img tag to next line and remove indent?

File: www/themes/Omicron/templates/viewnzb.tpl [hunk 3]
```diff
@@ -239,7 +239,13 @@
 														   id="guid{$release.guid}"></i> Send to
 														Queue
 														</button>{/if}
-													<p id="demo"></p>
+													{if !empty($cpurl) && !empty($cpapi)}
+														<button type="button"
+																class="btn btn-primary btn-sm btn-info btn-transparent couchsend">
```
Comment (b3rs3rk) [inline www/themes/Omicron/templates/viewnzb.tpl]: These few lines look weird

File: www/themes/Alpha/templates/upcoming.tpl [hunk 4]
```diff
@@ -55,9 +55,7 @@
 							{if $cpurl != '' && $cpapi != ''}
 								<a
 										class="sendtocouch"
```
Comment (b3rs3rk) [inline www/themes/Alpha/templates/upcoming.tpl]: Two indents here

**Comment by scottia** [comment]
Nice.  Looking forward to this one

**Comment by b3rs3rk** [comment]
Looks good.

**Pull request merged**